src/_help/hubs/hub-settings.md

```diff
@@ -16,26 +18,32 @@ The Custom Domain option allows you to keep your users within your brand experie
 > [Custom domain requires additional configuration on your end](/help/customization-options/custom-domains)
 {: .warning}
 
-## Description
+### Description
 
-This option allows you to add a description to your hub, and supports Markdown for enhanced text formatting. For instance, you can explain what this collection includes in terms of documentation, share important details, or even provide a contact for questions.
+This option allows you to add a description to your Hub, and supports Markdown for enhanced text formatting. For instance, you can explain what this collection includes in terms of documentation, share important details, or even provide a contact for questions.
 
 ![](/images/help/slug-hub.png)
 
-## Default Settings & Visual Customization
+## Hub Access
+
+Access management can be configured either at the [documentation level](/help/publish-documentation/documentation-access-management/) or at the Hub level.
+
+* If a Hub is public, all docs in that Hub are public, except the docs set to Private in the doc settings.
+* If a Hub is private, all docs in that Hub are private.
+
+## API Explorer
 
-These hub settings are applied by default to all the documentation it contains. From the Hub Settings tab, you can customize several general options.
+> This feature is currently in Beta test. If you are not a Beta tester, it may not appear in your Hub settings.
+{: .note}
 
-It is also possible to modify the settings of a documentation within a hub to customize it differently.
-  
-You can customize the visual appearance of your hub and its documentation by adding your logo and favicon directly from this section. By default, Bump.sh automatically generates a social media visual for you (when you share the URL of your hub or documentation on one of them). However, you can still choose your own image by uploading it at this step.
+Activating the feature on a hub will activate the feature for all docs in the hub.
 
-![](/images/help/hubs-customization.png)
+## Documentation listing
 
-You can also customize the general color scheme of your hub.
+This setting is useful when you have multiple API docs that you want to publish in a same Hub, and consider grouping them under categories.
 
-![](/images/help/color-scheme.png)
+Categories can be, for instance, groups of API that belong to a same business domain.
 
-Finally, you can choose to apply a general sorting parameter to all the documentations in your hub with the Group by operations (by path or tag) and Navigation options.
+Categories must be specified for each doc in the hub. When the "Group documentation by category" setting is selected, any documentation with no specified category will not be displayed in the Hub.
```

Comment:
While here, can mention that multiple categories option is possible, separated by a comma `","`

For example documentation with `"foo,bar"` will be visible in 2 categories, "foo" and "bar".

[Cf Slack message](https://itducks.slack.com/archives/C01DP38BRTL/p1744706623461049)